stable/mongodb/templates/statefulset-primary-rs.yaml

```diff
@@ -219,11 +219,16 @@ spec:
 {{ toYaml .Values.metrics.resources | indent 12 }}
 {{- end }}
       volumes:
-        {{- if  (.Files.Glob "files/docker-entrypoint-initdb.d/*[sh|js]") }}
+        {{- if (.Files.Glob "files/docker-entrypoint-initdb.d/*[sh|js]") }}
```

Comment:
You didn't update this line in the `deployment-standalone.yaml`, please update it.

Reply:
@botzill Please, it's an important change!

Reply:
Hi @pipko, can you elaborate on this? Do we need that space back?

Reply:
@botzill No, you didn't get it right: @juan131 and I mean that you forgot changes that you've done  in file statefulset-primary-rs.yaml with mounting the script or initConfigMap:

`
           {{- if  or (.Files.Glob "files/docker-entrypoint-initdb.d/*[sh|js]") (.Values.initConfigMap) }}
            - name: custom-init-scripts
              mountPath: /docker-entrypoint-initdb.d
            {{- end }}```
`